TurkishFilter/sections/specific.txt

```diff
@@ -431,7 +431,6 @@ habermotto.com##div[id^="kanews_widget_ads-"]
 memurlar.net##a[rel="sponsored"]
 maksatbilgi.com###tie-block_2320.media-overlay
 maksatbilgi.com##.theiaStickySidebar > .stream-item-widget
-||zf00.cloud/popmyshake.js
```

Comment:
Hi @gork7777 
Why did you remove this rule?

Reply:
@Sergey-Lyapin Doesn't exist anymore. At least I cannot reproduce again: https://github.com/AdguardTeam/AdguardFilters/commit/9a89de4016bf12ac590d2e3e594020cfab06adb1

Reply:
@gork7777 
But the script is still openable, it can be used somewhere else, it's better to keep the rule.